app/Console/Kernel.php

```diff
@@ -24,8 +25,9 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')
-        //          ->hourly();
+        $schedule->call(function () {
+            Availability::where('to', '<', Carbon::now())->delete();
```

Comment:
Might be worth adding an Eloquent query scope: https://laravel.com/docs/master/eloquent#local-scopes

Reply:
done